docs/source/tensor_view.rst

```diff
@@ -0,0 +1,74 @@
+.. currentmodule:: torch
+
+.. _tensor-view-doc:
+
+Tensor Views
+=================
+
+`Tensor` and `Storage` are two core concepts in PyTorch. `Tensor` is the main user facing data structure,
```

Comment:
I don't really understand why we need to explain storages.  Can't we explain using `_base` as a debugging tool instead?

Reply:
Well, `.storage()` is a public API, `._base` is hidden and should not be used by the users :D

Reply:
Maybe we keep the few sentences about storage and add an example of `_base` in the code?

Reply:
I meant the opposite :D 
You can describe it with base. But the code sample should only use public api: `.storage()`.

Reply:
Updated! I removed explicit mention of `Storage` data structure but use base tensor or general storage instead. Please let me know if this looks good!

Reply:
But the point was to avoid mentioning Storage, because this isn't an explanation of the internals of PyTorch, it was explaining what views are.

Actually, why do we need Storage at all?  Just compare the data_ptr of the tensor in your example.

Reply:
@gchanan It's not safe to call `tensor.data_ptr()` as `tensor`'s `data_ptr` points to address of the first element in tensor. Although it works in my simple `view` example, it'd fail when users do `tensor[1:]` where it's still view of `tensor`, but fails the check. 
I wanted to give a more general check that'll pass for all view ops in the example.